webapp/README.md

```diff
@@ -0,0 +1,51 @@
+# Docker monitoring - web dashboard.
+This readme describes how to configure and run web dashboard of docker monitoring app.
+
+1. Install all requirements from root requirements.txt
+    ```
+    pip install -r requirements.txt
+    ```
+2. Complete all steps from root readme and configure postgresql database.
+3. In `app.py` tune following line:
```

Comment:
Do we really need to modify source code file to up the application? Maybe we can move this configuration to some specific file? Json, taml of toml for example?

Reply:
Changed to json, readme will be changed in short time